### DISCUSSION TITLE
wait to close window

### DISCUSSION THREAD

**Discussion by Rirynn** [Akascape/CTkMessagebox#29] [Q&A] [answered]
Hi!

I have a script and I use MessageBox, how do I make the script wait for me to close the message box and then move on to the next operation, in tkinter message box it waits, but with this module it doesn't. Thanks!

**Comment by Akascape** [comment, marked as answer]
@Rirynn It will wait if you use the `.get()` method.
```python
msg = CTkMessagebox(app)
msg.get() # wait for the response
print("operation") # move to next operation after the response
```
